todo-list/src/App.vue

```diff
@@ -40,7 +40,8 @@ export default {
       tasks: [],
       id: 0,
       stateNum: 1,
-      state: ['完了','作業中']
+      state: ['完了','作業中'],
```

Comment:
`,`の後にはスペースを空けるようにしましょう。

```suggestion
      state: ['完了', '作業中'],
```